x-pack/legacy/plugins/apm/server/routes/__test__/routeFailures.test.ts

```diff
@@ -47,10 +47,8 @@ describe('route handlers should fail with a Boom error', () => {
     routes.forEach((route, i) => {
       test(`${route.method} ${route.path}"`, async () => {
         await expect(route.handler(mockReq)).rejects.toMatchObject({
-          message: 'request failed',
-          isBoom: true
+          message: 'request failed'
         });
-        expect(consoleErrorSpy).toHaveBeenCalledTimes(1);
```

Comment:
why are we removing this assertion?

Reply:
presumable because of https://github.com/elastic/kibana/pull/42252/commits/265cc3b26a9dc3d338a79d26fd749cef48fc0fed?

Reply:
Yes, exactly. Removing `defaultErrorHandler ` was perhaps a bit out of the scope for this PR so I can re-add it if you think. Although I don't think it provided us with much (and now the status code is correctly 500 instead of 400).